kevintdd/src/main/java/Item.java

```diff
@@ -0,0 +1,52 @@
+import java.math.BigDecimal;
```

Comment:
Look at proposed changes in attachment, tests could have to be adjusted to account for corrected totals
[Item.txt](https://github.com/KTBG7/tdd/files/4956449/Item.txt)